### DISCUSSION TITLE
Adicionando Dependências com Bower

### DISCUSSION THREAD

**Discussion by m0ntelo** [frontendbr/forum#435] [Perguntas e Respostas]
Quando adiciono uma dependência com o ela traz um pacote completo. Por exemplo eu adiciono bootstrap e ele instalou outros arquivos que não vou precisar para meu projeto.

Como adiciono somente o arquivo que eu necessito?

**Comment by cjpatoilo** [comment]
@WilsonMontelo Talvez esse pacote possa te ajudar se fizer uso de algum *Build System* como Grunt ou Gulp.

https://www.npmjs.com/package/main-bower-files

**Comment by kaisermann** [comment]
Você pode dar override no ["main" files](https://github.com/twbs/bootstrap/blob/v4-dev/bower.json#L17) do pacote tb.

**Comment by felipefialho** [comment]
@WilsonMontelo Conseguiu resolver?
